examples/plot_freesurfer_normalization.py

```diff
@@ -34,4 +34,5 @@
                               line_width=3, hemi=hemi)
     brain.contour_list[-1]["colorbar"].visible = False
 
-brain.show_view("dorsal")
+if not brain.patch_mode:
+    brain.show_view("dorsal")
```

Comment:
Instead of forcing people to write conditionals, in `brain.show_view` I would check to see if in `patch_mode` and if so, discard the azimuth/elevation/roll arguments (only pay attention to `distance`), and say that this is the behavior in the `show_view` docstring. If people *really* want to configure the view they can use `mlab` to do it.